src/style.css

```diff
@@ -0,0 +1,177 @@
+html {
```

Comment:
please split .css files: [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md#:~:text=You%20should%20create%20separate)
> 10. [FILE STRUCTURE] - You should create separate file for each BEM block